From dda7fed7d225da9242c767f2e548a4bf4d5fe022 Mon Sep 17 00:00:00 2001
From: Lennart Poettering <lennart@poettering.net>
Date: Tue, 6 Mar 2012 01:29:29 +0100
Subject: [PATCH] socket: rename 'broken' failure result to 'failed-permanent'
 on order not to introduce new terminology (cherry picked
 from commit 6bda96a0234e2d3d61bc36de99c19a9293721fef)

Conflicts:

	src/service.c
	src/socket.c
	src/socket.h
---
 src/service.c |    4 ++--
 src/socket.c  |    6 +++---
 src/socket.h  |    2 +-
 3 files changed, 6 insertions(+), 6 deletions(-)

diff --git a/src/service.c b/src/service.c
index 950e495..af83b6e 100644
--- a/src/service.c
+++ b/src/service.c
@@ -1375,7 +1375,7 @@ static int service_search_main_pid(Service *s) {
         return 0;
 }
 
-static void service_notify_sockets_dead(Service *s, bool broken) {
+static void service_notify_sockets_dead(Service *s, bool failed_permanent) {
         Iterator i;
         Unit *u;
 
@@ -1388,7 +1388,7 @@ static void service_notify_sockets_dead(Service *s, bool broken) {
 
         SET_FOREACH(u, s->meta.dependencies[UNIT_TRIGGERED_BY], i)
                 if (u->meta.type == UNIT_SOCKET)
-                        socket_notify_service_dead(SOCKET(u), broken);
+                        socket_notify_service_dead(SOCKET(u), failed_permanent);
 
         return;
 }
diff --git a/src/socket.c b/src/socket.c
index 4b85528..b40b7a1 100644
--- a/src/socket.c
+++ b/src/socket.c
@@ -2016,7 +2016,7 @@ int socket_collect_fds(Socket *s, int **fds, unsigned *n_fds) {
         return 0;
 }
 
-void socket_notify_service_dead(Socket *s, bool broken) {
+void socket_notify_service_dead(Socket *s, bool failed_permanent) {
         assert(s);
 
         /* The service is dead. Dang!
@@ -2025,8 +2025,8 @@ void socket_notify_service_dead(Socket *s, bool broken) {
          * services. */
 
         if (s->state == SOCKET_RUNNING) {
-                log_debug("%s got notified about service death (broken: %s)", s->meta.id, yes_no(broken));
-                if (broken)
+                log_debug("%s got notified about service death (failed permanently: %s)", s->meta.id, yes_no(failed_permanent));
+                if (failed_permanent)
                         socket_enter_stop_pre(s, false);
                 else
                         socket_enter_listening(s);
diff --git a/src/socket.h b/src/socket.h
index 31b9552..b172402 100644
--- a/src/socket.h
+++ b/src/socket.h
@@ -137,7 +137,7 @@ struct Socket {
 int socket_collect_fds(Socket *s, int **fds, unsigned *n_fds);
 
 /* Called from the service when it shut down */
-void socket_notify_service_dead(Socket *s, bool broken);
+void socket_notify_service_dead(Socket *s, bool failed_permanent);
 
 /* Called from the mount code figure out if a mount is a dependency of
  * any of the sockets of this socket */
